src/Swarm/Game/State.hs

```diff
@@ -617,7 +617,9 @@ scenarioToGameState scenario userSeed toRun g = do
       , _viewCenterRule = VCRobot baseID
       , _viewCenter = V2 0 0
       , _needsRedraw = False
-      , _replStatus = REPLDone
+      , _replStatus = case toRun of
```

Comment:
Maybe a short explanation would help here? This is how I understand it:
```suggestion
      , -- When starting base with the run flag, REPL status must be set to working,
        -- otherwise the store of definition cells is not saved (see #333)
        _replStatus = case toRun of
```

Reply:
Nice idea, thanks!